contrib/completions/meson.build

```diff
@@ -10,3 +10,11 @@ if get_option('fish-completions')
 	endif
 	install_data(fish_files, install_dir: fish_install_dir)
 endif
+
+bash_comp = dependency('bash-completion', required: false)
```

Comment:
Maybe only check the dep if `bash-completions` is enabled?

Reply:
I'm not familiar with meson so I did it the same way it is for fish. Should I do something like this?
```diff
@@ -1,7 +1,7 @@
-fish_comp = dependency('fish', required: false)
-
 if get_option('fish-completions')
        fish_files = files('fish/grim.fish')
+
+       fish_comp = dependency('fish', required: false)
        if fish_comp.found()
                fish_install_dir = fish_comp.get_pkgconfig_variable('completionsdir')
        else
@@ -11,10 +11,12 @@ if get_option('fish-completions')
        install_data(fish_files, install_dir: fish_install_dir)
 endif
 
-bash_comp = dependency('bash-completion', required: false)
 
-if get_option('bash-completions') and bash_comp.found()
-       bash_files = files('bash/grim.bash')
-       bash_install_dir = bash_comp.get_pkgconfig_variable('completionsdir')
-       install_data(bash_files, install_dir: bash_install_dir)
+if get_option('bash-completions')
+       bash_comp = dependency('bash-completion', required: false)
+       if bash_comp.found()
+               bash_files = files('bash/grim.bash')
+               bash_install_dir = bash_comp.get_pkgconfig_variable('completionsdir')
+               install_data(bash_files, install_dir: bash_install_dir)
+       endif
 endif
 ```

Reply:
Ah, yeah, the fish part seems off.

This diff looks good if you remove `required: false` for `bash-completion` (and the `if bash_comp.found()` condition): when the user asks to install completions, I'd rather fail hard when bash is missing rather than silently not install them.

Reply:
ah, whoops. Fixed it